src/platform/silabs/wifi/SiWx/WifiInterfaceImpl.cpp

```diff
@@ -555,29 +553,31 @@ CHIP_ERROR ResetCounters()
     return CHIP_NO_ERROR;
 }
 
-sl_status_t InitSiWxWifi(void)
+CHIP_ERROR InitWiFiStack(void)
 {
     sl_status_t status = SL_STATUS_OK;
 
     status = sl_net_init((sl_net_interface_t) SL_NET_WIFI_CLIENT_INTERFACE, &config, &wifi_client_context, nullptr);
-    VerifyOrReturnError(status == SL_STATUS_OK, status, ChipLogError(DeviceLayer, "sl_net_init failed: %lx", status));
+    VerifyOrReturnError(status == SL_STATUS_OK, CHIP_ERROR_UNINITIALIZED,
```

Comment:
This isn't the correct error code. We should check if there is a bit more code or use CHIP_ERROR_INTERNAL.